src/context.rs

```diff
@@ -346,7 +353,42 @@ impl Context {
         }
     }
 
-    /// Determine a TPM property, which is automatically cached if available
+    /// Determine a TPM property
+    ///
+    /// # Details
+    /// Returns the value of the provided `TpmProperty` if
+    /// the returned value is not supported by the library
+    /// but the opration was successful then `Ok(None)` is
+    /// returned.
+    /// The property is automatically cached if available.
+    ///
+    /// # Errors
+    /// If the TPM returns a value that is wrong when
+    /// its capabilities is being retrieved then a
+    /// `WrongValueFromTpm` is returned.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use tss_esapi::{Context, Tcti, constants::tags::PropertyTag};
+    /// use std::{env, str::FromStr};
+    ///
+    /// // Create context test TCTI used for internal testing.
+    /// let tcti = env::var("TEST_TCTI")
+    ///     .map_or_else(
+    ///         |_| {
+    ///             Tcti::Mssim(Default::default())
+    ///         },
+    ///         |v| {
+    ///             Tcti::from_str(&v).expect("Error parsing TEST_TCTI")
+    ///     });
+    /// let mut context = unsafe { Context::new(tcti).expect("Failed to create Context") };
```

Comment:
I would probably leave this out of the example shown in the docs ([with `#`](https://doc.rust-lang.org/rustdoc/documentation-tests.html#hiding-portions-of-the-example)) - it makes it too verbose otherwise. I'm not sure for the other examples how much you should leave in the visible example - e.g. does the session creation have something relevant in that sense for it to be included?

Reply:
The only reason it is there is because I had not found the flag to turn off testing the examples in the docmentation. So the CI would fail because I used the wrong TCTI. I really did not want it there. Now I have found the flag I was looking for which only makes sure that the examples are compiled but do not run them.

Reply:
This have been fixed now.